department-of-veterans-affairs/abd-vro
# Set up scheduled SecRel run #3031

## What was the problem?
We want the secrel action to run daily so we can monitor the results without manually triggering the event.

Associated tickets or Slack threads:
- #3031 

## How does this fix it?[^1]
This commit configures cron in github actions to run a secrel scan daily at 1100 UTC/0700 ET.  Pull request #3112 has tested the functionality.

## How to test this PR
- Step 1 view SecRel runs each morning after 0700 ET.


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
